lib/vagrant-service-manager/command.rb

```diff
@@ -8,14 +9,15 @@ module ServiceManager
     SCCLI_SERVICES = ['openshift', 'k8s']
     KUBE_NAMES = ['kubernetes', 'k8s']
     KUBE_SERVICES = [
-      "etcd",
-      "kube-apiserver",
-      "kube-controller-manager",
-      "kube-scheduler",
-      "kubelet",
-      "kube-proxy",
-      "docker"
+      'etcd', 'kube-apiserver', 'kube-controller-manager', 'kube-scheduler',
+      'kubelet', 'kube-proxy', 'docker'
     ]
```

Comment:
@LalatenduMohanty : Well, i thought let's group based on master and minion services which would be more readable in context of Kubernetes along with less lines